dist/fedora-infra/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM fedora:30
+
+ADD . /src
+
+# build: system utilities and libraries
+RUN dnf -y install g++ openssl-devel
+
+# build: pinned Rust stable toolchain (only use in case of temporary distro breakage)
+#ADD https://static.rust-lang.org/dist/rust-1.35.0-x86_64-unknown-linux-gnu.tar.gz rust.tar.gz
+#RUN tar -xf rust.tar.gz --strip 1 && ./install.sh
+
+# build: system Rust toolchain
+RUN dnf -y install rust cargo
+
+# build: graph-builder binary
+RUN cd /src && cargo build --release --bin graph-builder && mv /src/target/release/graph-builder /usr/local/bin/
```

Comment:
instead of using `cd` you can use [`WORKDIR`](https://docs.docker.com/engine/reference/builder/#workdir).